go.mod

```diff
@@ -49,7 +49,6 @@ require (
 
 require (
 	github.com/rs/zerolog v1.29.1
-	github.com/vektra/mockery/v2 v2.26.1
```

Comment:
Just remove? No bump up?

Reply:
It appears to have been erased by dependabot because it's not required by a dependency in the source code. it's also confirmed to be erased for `go mod tidy` as well. However, `vectra/mockery/v2` is required to run `make mockery`.

In order to prevent such packages erased from `go.mod`, I added the source including the import list for the build tools. This method is based on the following URL.

https://zenn.dev/kenghaya/articles/2f41736042e054

If you have a more suitable file name than `tools.go` to do this, please let us know.